Source/Engine/Core/Memory/BumpFastAllocation.h

```diff
@@ -0,0 +1,256 @@
+// Copyright (c) 2012-2024 Wojciech Figat. All rights reserved.
+
+#pragma once
+
+#include "Allocation.h"
+#include "Engine/Core/Log.h"
```

Comment:
Include `Log.h` below within `#if ASSERT_SAFE_BUMP_ALLOC` so header is ligher for compilation.